recipe/meta.yaml

```diff
@@ -9,27 +9,24 @@ source:
   sha256: 69d80bd33ce8f08e7cfeeb71cefddfc29cede25a85881e33dbae47576b96ed29
 
 build:
-  number: 2
-  # no pytorch on windows in conda-forge, see
-  # https://github.com/conda-forge/pytorch-cpu-feedstock/issues/32
+  number: 0
+  skip: true  # [py<37]
   skip: true  # [win]
+  skip: true  # [ppc64le and py>=311]
   script:
-    - export BUILD_S3=TRUE
-    - export CMAKE_GENERATOR="Ninja"
     - export USE_SYSTEM_LIBS=TRUE
     - {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
-    - ninja
   host:
     - pip
     - python
-    - pybind11
-    - aws-sdk-cpp
-    - pytorch
+    - pytorch 1.13.1
+    - setuptools
+    - wheel
   run:
     - python
     - requests
```

Comment:
I think `torchdata` works only with `pytorch 1.13.1` in `run`, see https://github.com/pytorch/data/commit/9eda5be3c6c6679ddbfd789f6e15928aa85e4833